Infra AWS Migrated Templates Group 1

lroyce opened the pull request:
# Summary
5 New Relic AWS infra-dashboard-templates have been migrated to New Relic IO. Json, screenshots were included for each, following the contribution guidelines. Quickstarts were updated where necessary or created. Correct AWS logo was included with quickstart config. Also updated names where needed, some were AWS services while others were Amazon services. Also combined templates where it made sense into tabbed dashboards.

The 5 templates are:

1. `newrelic-quickstart/dashboards/aws-alb/aws-alb.json`
2. `newrelic-quickstart/dashboards/amazon-api-gateway/amazon-api-gateway.json`: Contains amazon-api-gateway-apis and amazon-api-gateway-stages
3. `newrelic-quickstart/dashboards/aws-billing-budgets/aws-billing.json`: Contains aws-billing-budgets and aws-billing-costs
5. `newrelic-quickstart/dashboards/amazon-cloudfront/amazon-cloudfront.json`
6. `newrelic-quickstart/dashboards/aws-auto-scaling/aws-auto-scaling.json`

<!-- DON'T DELETE THIS SECTION BELOW IF SUBMITTING A NEW QUICKSTART -->
## Pre merge checklist

<!-- This CHECKLIST SHOULD BE SUBMITTED FULLY COMPLETE WITH THE PR. IF NOT COMPLETE
THE PR REVIEW WILL BE DELAYED -->

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

**Aws-alb**
![aws-alb1](https://user-images.githubusercontent.com/33464917/190000393-b8956930-dacf-453b-b58f-d9f2fd29f55c.png)

![aws-alb2](https://user-images.githubusercontent.com/33464917/190000519-e14d4410-a977-4c9b-a996-edecff00a6e5.png)

**aws-apigateway-apis**
![aws-apigateway-apis](https://user-images.githubusercontent.com/33464917/190000576-c20dd89c-8c4e-4ade-9095-e5359ad3ebb7.png)


**aws-apigateway-stages**
![aws-apigateway-stages1](https://user-images.githubusercontent.com/33464917/190000621-32e1d170-35f7-45bb-9cd4-05ba87965f9d.png)


![aws-apigateway-stages2](https://user-images.githubusercontent.com/33464917/190000649-68420e2c-571e-43bc-bbb0-0e592d99d57c.png)


**aws-appsync**
![aws-appsync](https://user-images.githubusercontent.com/33464917/190000706-1a8ad4b8-078b-4b9b-aee0-8e33cc7dcf40.png)


**aws-athena**
![aws-athena](https://user-images.githubusercontent.com/33464917/190000732-60759799-b67a-41ab-bc79-3c100a92a6bb.png)


**aws-auto-scaling**
![aws-autoscaling1](https://user-images.githubusercontent.com/33464917/190000764-1e08d19b-d71f-4ebd-a546-5d1e5244cc7e.png)

![aws-autoscaling2](https://user-images.githubusercontent.com/33464917/190000778-c9df896a-fb95-44f7-b5e5-68e867fe1e2c.png)


**aws-cloudfront-migrated**
![aws-cloudfront](https://user-images.githubusercontent.com/33464917/190000926-14c8443a-b038-444f-b830-c5dcffa2ff3d.png)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/newrelic/newrelic-quickstarts?pullRequest=1362) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/newrelic/newrelic-quickstarts?pullRequest=1362) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: lroyce<br/>:x: Nicky Duvieilh<br/><hr/>**Nicky Duvieilh** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/newrelic/newrelic-quickstarts?pullRequest=1362) it.</sub>

lroyce commented:
This PR includes 9 migrated [infra-dashboard-templates](https://source.datanerd.us/infrastructure/infra-dashboard-templates) and their quickstart configs. 

Dashboard / Quickstart
1. `newrelic-quickstart/dashboards/aws-alb` / `newrelic-quickstart/aws/aws-nlb-alb/`
2. `newrelic-quickstart/dashboards/aws-apigateway-apis` / `newrelic-quickstart/aws/aws-apigateway/`
3. `newrelic-quickstart/dashboards/aws-apigatway-stages` / `newrelic-quickstart/aws/aws-apigateway/`
4. `newrelic-quickstart/dashboards/aws-appsync` / `newrelic-quickstart/aws/aws-appsync/`
5. `newrelic-quickstart/dashboards/aws-athena` / `newrelic-quickstart/aws/aws-athena/`
6. `newrelic-quickstart/dashboards/aws-billing-budgets` / `newrelic-quickstart/aws/aws-billing/`
7. `newrelic-quickstart/dashboards/aws-billing-costs` / `newrelic-quickstart/aws/aws-billing/`
8. `newrelic-quickstart/dashboards/aws-cloudfront-migrated` / `newrelic-quickstart/aws/aws-cloudfront/`
9. `newrelic-quickstart/dashboards/aws-autoscaling` / `newrelic-quickstart/aws/aws-auto-scaling/`

There are a few templates that were migrated and are duplicates of currently existing IO templates, we are waiting to hear which should remain in the PR. These templates are: 

infra-dashboard-template / IO template
1. `newrelic-quickstart/dashboards/aws-alb` / `newrelic-quickstart/dashboards/alb/`
      - the existing IO template `alb.json` seems formatted incorrectly
      - the migrated infra template has 2 more widgets and accompanying queries.
2. `newrelic-quickstart/dashboards/aws-billing-budgets` *AND* `newrelic-quickstart/dashboards/aws-billing-costs` / `newrelic-quickstart/aws-billing/`
      - the existing IO template `aws-billing` seems to combine both migrated templates `aws-billing-budgets.json` and `aws-billing-costs.json`
3. `newrelic-quickstart/dashboards/aws-cloudfront-migrated` / `newrelic-quickstart/dashboards/aws-cloudfront/`

aswanson-nr commented on quickstarts/aws/aws-athena/config.yml:
```diff
@@ -1,7 +1,7 @@
 id: 5f3d35d7-668b-41f3-98d7-ccc93a50933c
-slug: amazon-athena
+slug: aws-athena
```
In general, we do not change the slug on quickstarts because that is used as part of the url pathing to a particular quickstart. If we really need this change, let's sync on ways to do this without breaking links for people.

lroyce replied:
I'll change the slug back and I've let everyone know to not change that. 
I'll also try to find accounts with data for `dashboards/aws-appsync` and `dashboards/aws-athena` though regarding `dashboards/aws-auto-scaling`, the screenshots do seem to have data in them, can you confirm?

lroyce replied:
Unfortunately for `dashboards/aws-appsync` and `dashboards/aws-athena`, I've not been able to find any non-0 data.

aswanson-nr reviewed (changes requested):
Hey @lroyce, 
These are looking good! I only saw one issue around the `aws-athena` change and a few screenshots that need to be updated, but other than that it looks good to go!

1. `dashboards/aws-appsync` doesn't have any data in the screenshot
2. `dashboards/aws-athena` doesn't have any data in the screenshot
3. `dashboards/aws-auto-scaling` doesn't have any data in the screenshots

aswanson-nr commented:
Also, just FYI @nduvieilh will need to sign the CLA before we can merge this.

aswanson-nr reviewed (commented):
This group is looking good to go. @lroyce Would you like me to more forward and merge?

lroyce commented:
> This group is looking good to go. @lroyce Would you like me to more forward and merge?

@aswanson-nr I think we are good to merge. I removed `aws-appsync` and `amazon-athena`; updated both respective configs to remove the dashboards properties for now. 

Also combined `aws-billing-costs` and `aws-billing-budgets` into a single tabbed dashboard. Did the same with `amazon-api-gateway` (formerly known as `aws-api-gateway`) which combines `amazon-api-gateway-apis` and `amazon-api-gateway-stages`.
